doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 
 <a name="1.3"></a>
 ## Operational and Administrative Terminology
 
-- **Cloud service user**: Natural person, or entity acting on their behalf, associated with a cloud service customer that uses cloud services.
+- **Cloud service user:** Natural person, or entity acting on their behalf, associated with a cloud service customer that uses cloud services.
 >_*Note*_ Examples of such entities include devices and applications.
-- **Tenant**: One or more cloud service users sharing access to a set of physical and virtual resources ([ITU](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-Y.3500-201408-I!!PDF-E&type=items)).
->_*Note*_ Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
-- **Tenant (Internal) Networks**: Virtual networks that are internal to tenant instances.
-- **Multi-tenancy**: feature where physical, virtual or service resources are allocated in such a way that multiple tenants and their computations and data are isolated from and inaccessible by each another. ([ETSI](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/003/01.04.01_60/gs_nfv003v010401p.pdf))
-- **External Network**: External networks provide network connectivity for a cloud infrastructure tenant to resources outside of the tenant space.
-- **Quota**: An imposed upper limit on specific types of resources, usually used to prevent excessive resource consumption in the **VIM** by a given consumer (tenant).
-- **Resource pool**: A logical grouping of cloud infrastructure hardware and software resources. A resource pool can be based on a certain resource type (for example, compute, storage, network) or a combination of resource types. An **Cloud Infrastructure** resource can be part of none, one or more resource pools.
-- **Compute Node**: An abstract definition of a server.
+
+- **Compute Node:** An abstract definition of a server.
 >_*Note:*_ A compute node can refer to a set of hardware and software that support the VMs or Containers running on it.
-- **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
+
+- **External Network:** External networks provide network connectivity for a cloud infrastructure tenant to resources outside of the tenant space.
+
+- **Multi-tenancy:** feature where physical, virtual or service resources are allocated in such a way that multiple tenants and their computations and data are isolated from and inaccessible by each another. ([ETSI](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/003/01.04.01_60/gs_nfv003v010401p.pdf))
+
+- **Quota:** An imposed upper limit on specific types of resources, usually used to prevent excessive resource consumption in the **VIM** by a given consumer (tenant).
```

Comment:
to make it generic for both VM and containers
```suggestion
- **Quota:** An imposed upper limit on specific types of resources, usually used to prevent excessive resource consumption by a given consumer (tenant, VM, container).
```